Sources/PostgresNIO/New/PSQL+JSON.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 02.02.21.
+//
+
+import class Foundation.JSONEncoder
```

Comment:
Ditch Xcode template headers.
```suggestion
import class Foundation.JSONEncoder
```